HyperPlatform/vmm.h

```diff
@@ -50,9 +50,9 @@ struct KtrapFrameX86 {
   ULONG sp;  //!< Called HardwareEsp in _KTRAP_FRAME
   ULONG reserved3[5];
 };
-static_assert(sizeof(KtrapFrameX86) == 0x8c);
-static_assert(FIELD_OFFSET(KtrapFrameX86, ip) == 0x68);
-static_assert(FIELD_OFFSET(KtrapFrameX86, sp) == 0x74);
+static_assert(sizeof(KtrapFrameX86) == 0x8c, "");
```

Comment:
Please put some helpful error messages. I might say "structure size mismatch".  

It did not compile on VS2015 despite that `<LanguageStandard>stdcpp17</LanguageStandard>` did not break build, correct?

Reply:
vs2015 no longer works after installing VS2019 / WDK 18362.  Please advise.

Reply:
Can you elaborate how it does not work? Do you mean compile error (if so, logs would be helpful)? 

If it is more like error due to corrupt environment because of having multiple VS versions, having separate VMs might be help.

Reply:
Yes, I can explain.   

Driver vs projects (from before I installed VS 2019) will no longer open.  I can still open the solution and the user-mode application vs projects.  

VS 2015 repair did not fix the problem.  The repair broke the ml64.exe in VS 2019.  

Good idea - I will put VS2019 on this box an use another for VS 2015.

Reply:
Sorry to hear that :(